dataGenerator.js

```diff
@@ -0,0 +1,74 @@
+const faker = require('faker');
+const fs = require('fs');
+// const bson = require('bson');
+// const loremHipsum = require('lorem-hipsum');
+
+let dataList = [];
+
+//number of entries to generate -- defaults to 100k if not provided as arg
+const entryQty = process.argv[2] || 100000;
```

Comment:
Using the command line argument looks really nice. Just wondered if you created 100 files, next you would need to seed those 100 files to your databases. Do you think the scripts' sounds performance efficient?